App/Components/PhotoGrid.tsx

```diff
@@ -78,6 +78,8 @@ class PhotoGrid extends React.Component<ScreenProps & DispatchProps & Navigation
     const { item } = row
     switch (item.type) {
       case 'photo':
+        const files = item.photo.files
+        const fileIndex = files && files.length > 0 && files[0].index ? files[0].index : 0
```

Comment:
not sure how this could happen, but it matches the pattern of some crashes we were seeing